docs/reference/intro-to-semver.md

```diff
@@ -10,18 +10,18 @@ Version Promiscuity is the opposite problem, **JsonLibrary** releases _v1.1.0_ w
 
 SemVer introduces conventions about breaking changes into our version numbers so we can safely upgrade dependencies without fear of unexpected breaking changes while still allowing us to upgrade downstream libraries to get new features and bug fixes. The convention is quite simple.
 
-{major}.{minor}.{patch}-{tag}+{buildmetadata}  
-{major} = Only incremented if the release has breaking changes (includes bug fixes which have breaking behavioural changes
-{minor} = Incremented if the release has new non-breaking features
-{patch} = Incremented if the release only contains non-breaking bug fixes
-{tag} = Optional -{tag} denotes a pre-release of the version preceeding
-{buildmetadata} = Options +{buildmetadata} contains additional information about the version, but DOES NOT AFFECT the semantic version preceding it.
+**{major}.{minor}.{patch}-{tag}+{buildmetadata}**  
```

Comment:
Ignore, there are the spaces, the difftool wasn't showing them unlike the other lines